activesupport/lib/active_support/ordered_options.rb

```diff
@@ -3,7 +3,8 @@
 require "active_support/core_ext/object/blank"
 
 module ActiveSupport
-  # Usually key value pairs are handled something like this:
+  # +OrderedOptions+ extends the +Hash+ object to make it easier to create or access keys.
+  # Usually we create key value pairs like:
```

Comment:
"Easier" might be debatable here.  What if we were more explicit?

```suggestion
  # +OrderedOptions+ inherits from +Hash+ and provides dynamic accessor methods.
  #
  # With a +Hash+, key-value pairs are typically managed like this:
```